in.cinny.Cinny.yml

```diff
@@ -0,0 +1,37 @@
+id: in.cinny.Cinny
+runtime: org.gnome.Platform
```

Comment:
Does it actually need GNOME runtime or freedesktop would suffice here?

Reply:
It builds with freedesktop too. Are there benefits of using that over gnome one?

Reply:
smaller runtime is always better.